src/components/media_control/public/media-control.scss

```diff
@@ -182,14 +182,14 @@
         height: 100%;
         font-size: 20px;
 
-        &:before { content:"\\e001"; }
+        &:before { content:"\e001"; }
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 3<br>Colon after property should be followed by one space